gateway/tests/api/test_models.py

```diff
@@ -20,8 +23,123 @@ def test_job_is_terminal_state(self):
         job.status = Job.STOPPED
         self.assertTrue(job.in_terminal_state())
 
+        job.status = Job.QUEUED
+        self.assertFalse(job.in_terminal_state())
+
         job.status = Job.FAILED
         self.assertTrue(job.in_terminal_state())
 
         job.status = Job.SUCCEEDED
         self.assertTrue(job.in_terminal_state())
+
+
+class TestProgramSignals(TransactionTestCase):
+    def test_program_instances_signal(self):
```

Comment:
I don't know why this linter is not blaming here but would it be possible to add a little description of what the test does, @avilches ? 🙏